mailhandler/spameater

```diff
@@ -652,15 +682,15 @@ $config->debug('actually not... the account is disabled.');
             $updatelog = ! $util->hasFeature('DONOTLOGHIDDEN',$Features);
 
 ##End ADDED Syskoll 2008-03-30
-            &sendMail($config,$useXHeader,$SpamEmail,\$msg,
+            &sendMail($config,$useXHeader,\@rcpts,\$msg,
```

Comment:
Based on the previous code since $SpamEmail seems to be redeclared under the new name.
```suggestion
            &sendMail($config,$useXHeader,\@spamrcpt,\$msg,
```